src/kw_remote.sh

```diff
@@ -210,6 +215,18 @@ function rename_remote()
   fi
 }
 
+function list_remotes()
+{
+  # check if the remote config file exists
+  if [ -f "$local_remote_config_file" ]; then
+    cat $local_remote_config_file
+    exit 0
+  else
+    echo "There are no remotes"
```

Comment:
Please, don't use `echo` refer to https://kworkflow.org/content/codingstyle.html#use-printf-instead-of-echo